src/nifler/bridge.nim

```diff
@@ -658,13 +658,18 @@ template bench(task, body) =
   else:
     body
 
+proc errorHandler(conf: ConfigRef; info: TLineInfo; msg: TMsgKind; arg: string) =
+  msgs.message(conf, info, msg, arg)
+  quit QuitFailure
```

Comment:
but then this simply quits after the first error. Instead nifler should `quit 1` if the errorCounter > 0